### ISSUE TITLE
BLOCKER - The page’s second column content is hidden in print stylesheets.

### ISSUE THREAD

**Issue by sabderemane** [django/djangoproject.com#1410]
> Blocker: Critical issue. Certain users cannot perceive or operate this, with no workaround available.

This makes it impossible for users of magnifiers to access this content ([22:00](https://www.youtube.com/watch?v=dAiMPTOMh1U&t=1320s)).

**Comment by Varlight** [comment]
@sabderemane I would like to work on this issue

**Comment by sabderemane** [comment]
Hi @Varlight ! Feel free to work on the issue, no need to be assigned 🤗

**Comment by akshayrajput12** [comment]
@sabderemane  i would like to contribute on this issue

**Comment by fenil-ptl** [comment]
starting contribute

**Comment by notty0** [comment]
Hey  today i resolve this issue

**Comment by dikshasaraswa** [comment]
@sabderemane I would like to contribute on this issue.

**Comment by sabderemane** [comment]
Hi @Om15102003 no need to be assigned to work on this. @dikshasaraswa if @Om15102003  is working on this issue, I suggest to check another issue.
